status_interfaces - WIP for discussion

sbeaver-netgate opened the pull request:
This is one way to present the information without the 600 lines of
HTML that were in the file.

Comments or suggestions?

sbeaver-netgate committed:
status_interfaces - WIP for discussion

This is one way to present the information without the 600 lines of
HTML that were in the file.

Comments or suggestions?

SjonHortensius commented:
I think you did a great job reducing duplicate code, the `dl` is also nice. I do see some room for improvement:
- let showDef always perform htmlspecialchars over $def and remove that it in all calls
- same for showDefBtn/$term
- fix indenting on line 96-100
- there's a redundant `</div>` at the end, `<div class="panel-body"><dl class="dl-horizontal">` should have one less leading tab

sbeaver-netgate committed:
status_interfaces detailed changes per SH

Useful suggestions. Thank you.

htmlspecialchars() moved to showDef[Btn] functions
Minor formatting changes
